libs/@local/hash-subgraph/src/temp/map-vertices.ts

```diff
@@ -83,13 +83,7 @@ const mapEntityType = (entityType: EntityTypeGraphApi): EntityType => {
 export const mapOntologyMetadata = (
   metadata: OntologyElementMetadataGraphApi,
 ): OntologyElementMetadata => {
-  return {
-    ...metadata,
-    recordId: {
-      baseUri: metadata.editionId.baseId,
-      version: metadata.editionId.version,
-    },
-  };
+  return metadata;
```

Comment:
Just flagging for posterity that these are left here purposefully by Tim as directed by me. I intend to remove these in a follow-up PR, but there will be a few more maps we need to do around entity versioning and it'll be good to have this boilerplate in place to put some things in in the interim